nslsii/detectors/maia.py

```diff
@@ -0,0 +1,873 @@
+from collections import OrderedDict
+import time as ttime
+
+import numpy as np
+
+from ophyd import Device, Component as FldCpt, EpicsSignal, EpicsSignalRO
+from ophyd.status import DeviceStatus
+from ophyd.signal import Signal
+
+
+from recordwhat import RecordBase, _register_record_type, FieldComponent as Cpt
```

Comment:
Mistaken `as Cpt` here instead of above?

```suggestion
from recordwhat import RecordBase, _register_record_type, FieldComponent as FldCpt
```

I'd also just wholesale nab `AiRecord` and all from recordwhat if I were you 🤷‍♂️

Reply:
🤦🏻 fixed.